### ISSUE TITLE
Compresstion

### ISSUE THREAD

**Issue by RichardPar** [v3l0c1r4pt0r/lkv-wiki#1]
Below is some code for LZSS compression - written by ITE for Apple.. perhaps relevant. The format seems to be used for Compressing images (maybe the NO SIGNAL image that is streamed)

https://opensource.apple.com/source/xnu/xnu-3789.21.4/osfmk/console/video_console.c.auto.html

**Comment by v3l0c1r4pt0r** [comment]
Yep, seems like `vc_display_lzss_icon` function contains pretty standard LZSS implementation. Now some check need to be done, if this matches somehow blobs in firmware image.

**Comment by RichardPar** [comment]
looks the same as...

http://dev.gameres.com/Program/Other/LZSS.C

http://www.metroid2002.com/retromodding/wiki/LZSS_Compression
